doc/devel/coding_guide.rst

```diff
@@ -136,7 +136,7 @@ Milestones
   * *New features and API changes* are milestoned for the next minor release
     ``v3.N.0``.
 
-  * *Bugfixes and docstring changes* are milestoned for the next patch
+  * *Bugfixes, tests, and docstring changes* are milestoned for the next patch
```

Comment:
Not sure whether we should clarify "tests for existing code". Of course I cannot milestone tests for a new feature. Nevertheless it might be reasonable to very explicit here.

Reply:
Fine by me.